### PR TITLE
Added documentation for optoma_projector

### PR DESCRIPTION
**Pull request by wesleylima**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/optoma_projector.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+The `optoma_projector` switch platform allows you to control the state of a network enabled Optoma projector.
+
+## {% linkable_title Configuration %}
+
+To use your Optoma Projector in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
```
Comment (klaasnicolaas) [inline source/_components/optoma_projector.markdown:26]: Only required items in the first example

File: source/_components/optoma_projector.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: optoma.png
+ha_category:
+  - Multimedia
+ha_iot_class: Local Polling
+ha_release: 0.94
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/optoma_projector.markdown]: `redirect_from` is only for existing platform pages that will be merged.

File: source/_components/optoma_projector.markdown [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+layout: page
+title: "Optoma Projector Switch"
+description: "Instructions on how to integrate Optoma Projectors into Home Assistant."
+date: 2019-05-14 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: optoma.png
+ha_category:
```
Comment (klaasnicolaas) [inline source/_components/optoma_projector.markdown:11]: missing `switch` category?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by wesleylima** [comment]
Changes made

File: source/_components/optoma_projector.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## {% linkable_title Configuration %}
+
+To use your Optoma Projector in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: optoma_projector
+    host: 192.168.1.111
```
Comment (klaasnicolaas) [inline source/_components/optoma_projector.markdown]:
```suggestion
    host: IP_ADDRESS
```

File: source/_components/optoma_projector.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  description: The name to use when displaying this switch.
+  required: false
+  type: string
+host:
+  description: The network path to the projector.
+  required: true
+  type: string
+username:
+  description: Username to log into the Optoma web ui (default is admin).
+  required: false
```
Comment (klaasnicolaas) [inline source/_components/optoma_projector.markdown:43]: you can add: (also with password)
```
default: admin
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
A small and friendly notification @wesleylima! 👋 

Are you planning on moving forward on this PR? 
I didn't see any activity recently, while there are still things to do. I hope you consider picking it up soon! 👍

**Comment by wesleylima** [comment]
Just committed suggest changes :)

File: source/_components/optoma_projector.markdown [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: optoma.png
+ha_category:
+  - Multimedia
+  - switch
+ha_iot_class: Local Polling
+ha_release: 0.94
```
Comment (klaasnicolaas) [inline source/_components/optoma_projector.markdown]:
```suggestion
ha_release: 0.96
```

File: source/images/supported_brands/QuantitySoldBuilder.php [hunk 7]
```diff
@@ -0,0 +1,58 @@
+<?php
```
Comment (klaasnicolaas) [inline source/images/supported_brands/QuantitySoldBuilder.php]: What is the purpose of this file?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/images/supported_brands/QuantitySoldBuilder.php (hunk 7 above)
Comment (wesleylima) [reply source/images/supported_brands/QuantitySoldBuilder.php]: Whoops, must have gotten accidentally dragged in there from somewhere else. Deleted.

**Review by klaasnicolaas — APPROVED** [review]

**Comment by frenck** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**